README.md

```diff
@@ -21,15 +21,21 @@ This image is available on Docker Hub: [lequal/sonarqube](https://hub.docker.com
 
 Since inception, this image has been designed to be used in production. Thus, leaving the default admin password (namely "admin") will never be an option. To this extent, a new password for the admin account must be given by setting the environment variable `SONARQUBE_ADMIN_PASSWORD`.
```

Comment:
Is it really more secure to provide an admin password through command line and store it as environment variable?

In case the user use an external database, a different admin password should already be set up, isn't it?

Reply:
As discussed, it will be stated in the README that the container won't run unless a password is set for the admin account by this mean. Everything is more secure than an admin password equal to "admin". Moreover, the admin password does not appear in the image.

The password for the database is different from the password on SonarQube.